pom.xml

```diff
@@ -10,7 +10,7 @@
   <artifactId>plugin</artifactId>
   <!-- Baseline Jenkins version you use to build and test the plugin. Users 
    must have this version or newer to run. -->
-  <version>2.8</version>
+  <version>1.609.3</version>
```

Comment:
No, the comment is correct. If your define the pom.xml like that, you will build the plugin against version 2.8, even you can test it against lower version. That is exactly what we shouldn't do. We need to early detect a leaked API usage of something what isn't supported in 1.609.3. So the easiest way how to achieve that is to depend on the lower version of Jenkins if there isn't a requirement for upper one.